tests/integration/pilot/mirror_test.go

```diff
@@ -163,7 +170,7 @@ func runMirrorTest(t *testing.T, options mirrorTestOptions) {
 func sendTrafficMirror(from, to echo.Instance, proto protocol.Instance, testID string) error {
 	options := echo.CallOptions{
 		Target:   to,
-		Count:    50,
+		Count:    250,
```

Comment:
nit: do we need this many requests?

Reply:
Still going to bump it to 100, but yeah this is overkill. We want to bump it so that when we have 5 clusters in CI we can validate cross-cluster load balancing. Same goes for other call counts, which I'll also reduce.